zips/zip-0207.rst

```diff
@@ -59,7 +59,7 @@ requirements for the Zcash Development Fund, the ZIP was reintroduced for that
 purpose in the Canopy upgrade in order to reuse specification, analysis, and
 implementation effort.
 
-As of NU6, ZIP 1015 [#zip-1015]_ directs part of the block reward to a reserve,
+As of NU6, ZIP 1015 [#zip-1015]_ directs part of the block subsidy to a reserve,
```

Comment:
Should we use "issuance" rather than "subsidy"?

Reply:
No, block subsidy has a well defined meaning identical to the Bitcoin meaning. Also that change would be disruptive in many places in the protocol spec.

If and when ZIP 234 activates, it's still correct to refer to this as block subsidy; it's just a different value. If ZSAs activate then that has its own definition of issuance (which we need to deconflict).